code/modules/mob/living/silicon/silicon.dm

```diff
@@ -215,6 +215,8 @@
 		statelaws()
 
 	if (href_list["printlawtext"]) // this is kinda backwards
+		if (href_list["dead"] && (!isdead(usr) && !usr.client.holder)) // do not print deadchat law notice if the user is now alive
+			return
```

Comment:
>it's just to stop obvious abuse.

Fair enough. In that case I think there should be a warning added so it does not act the same as it would for a purged AI, and is a subtle reminder not to use that info IC even if they know how to read the href, and prevents them thinking there is a bug stopping output.

```suggestion
			to_chat(usr, "<span class='warning'>You cannot view law changes while alive.</span>")
			return
```

Reply:
I thought a notice would be smart but I did not follow through. 
This is a good idea, a notice is now added.